### PR TITLE
feat: imported and called the emoteListener

### LINKED ISSUE #1075: [FEAT] - Emote-only on Message Edit
### What feature would you like to see?

We need to mount the emote only listener to the message edit event. To do this, we should import the `emoteListener` in the [`messageUpdate` file](https://github.com/BeccaLyria/discord-bot/blob/main/src/events/messageEvents/messageUpdate.ts). Then, on line 75, we should call it with `await emoteListener(Becca, message, serverConfig)` (below the other listeners).

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by divyakelaskar**
# Pull Request

## Description

Mounts the emote only listener to the message edit event.

## Related Issue

Closes #1075

## Documentation

For _any_ version updates, please verify if the [documentation page](https://www.beccalyria.com/discord-documentation) needs an update. If it does, please [create an issue there](https://github.com/BeccaLyria/discord-documentation/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=update.md&title=%5BUPDATE%5D) to ensure it is not forgotten.

- [x] My contribution does NOT require a documentation update.
- [ ] My contribution DOES require a documentation update.

### CODE REVIEW

File: src/events/messageEvents/messageUpdate.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { triggerListener } from "../../listeners/triggerListener";
 import { sendLogEmbed } from "../../modules/guild/sendLogEmbed";
 import { getSettings } from "../../modules/settings/getSettings";
 import { beccaErrorHandler } from "../../utils/beccaErrorHandler";
+import { emoteListener } from "../../listeners/emoteListener";
```
Comment (nhcarrigan) [inline src/events/messageEvents/messageUpdate.ts:11]: This import will need to go between the `automodListener` and `sassListener` imports. 🙂

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

**Comment by divyakelaskar** [comment]
> This import will need to go between the `automodListener` and `sassListener` imports. 🙂

Why does the order of import statements matter ?

**Comment by nhcarrigan** [comment]
It has to do with the linter configuration - having a specific import order can help avoid circular dependency issues, and also keeps things a bit cleaner. 🙂

**Review by nhcarrigan — APPROVED** [review]
This looks good to me!

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Comment by divyakelaskar** [comment]
Thanks for clarifying my question earlier.

And yeah it feels great to contribute (even a tiny bit) to Becca, as I have seen her moderating so many communities efficiently.

Also, congratulations to Becca getting verified 🎉 

Great work, both of you !

**Pull request merged**